### PR TITLE
Fix : Add logo for dark theme

### LINKED ISSUE #600: PyMech logo difficult to see in dark mode
In dark mode on GitHub, the PyMechanical logo in the README is difficult to see. Let's make a dark mode logo to fix this.

URL: https://mechanical.docs.pyansys.com/version/dev/index.html

darkmode_logo

### PR DESCRIPTION
**Pull request by dipinknair**
Fixes #600

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,5 +1,5 @@
-.. image:: doc/source/images/pymechanical-logo-light.png
-   :alt: PyMechanical
+.. image:: https://raw.githubusercontent.com/ansys/pymechanical/main/doc/source/_static/logo/pymechanical-logo-light.png
```
Comment (RobPasMue) [inline README.rst:1]: I wouldn't use the public facing URL - use the local path

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -27,7 +27,7 @@ repos:
   - id: codespell
 
 - repo: https://github.com/ansys/pre-commit-hooks
-  rev: v0.2.8
+  rev: v0.2.9
```
Comment (RobPasMue) [inline .pre-commit-config.yaml:30]: Do this in a different PR. Let's try to keep PRs as clean as possible.

**Review by RobPasMue — CHANGES_REQUESTED** [review]

File: README.rst (hunk 1 above)
Comment (dipinknair) [reply README.rst:1]: okay. but if we use the local path, then it wont show up in pypi page ➡️  https://pypi.org/project/ansys-mechanical-core/

if that's okay , I can update with local path.

Comment (RobPasMue) [reply README.rst:1]: Ah got it. Okay, let's leave it like this then

**Review by RobPasMue — APPROVED** [review]

File: .github/workflows/ci_cd.yml [hunk 3]
```diff
@@ -236,7 +236,7 @@ jobs:
   embedding-tests:
     name: Embedding testing and coverage
     runs-on: ubuntu-latest
-    timeout-minutes: 10
+    timeout-minutes: 15
```
Comment (RobPasMue) [inline .github/workflows/ci_cd.yml:239]: Why are we changing this in this PR?

**Review by RobPasMue — CHANGES_REQUESTED** [review]

File: .github/workflows/ci_cd.yml (hunk 3 above)
Comment (dipinknair) [reply .github/workflows/ci_cd.yml:239]: I am just testing it to see what the failure is, currently other embedding tests are passing. some of them automatically failing because of the 10 mins limit.

**Review by RobPasMue — APPROVED** [review]
